pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
I'm not sure I understand the question, but I haven't read over the prefix parameter code much yet. Isn't F* the prefix parameter?  Or, do prefix parameters necessarily count 1, 2, 3, .. instead of 0, 1, 2, 3, ..?  Looking at the code below, it seems that only F2 and above are prefixParameters.  Why is that?